bot/cogs/commands/owner.py

```diff
@@ -290,6 +290,26 @@ def format_row(idx_row) -> str:
             f"**Command Streaks** (last week)\n```md\n## d  h  m  s  | user id             | name\n{formatted_rows}\n```"
         )
 
+    @commands.command(
+        name="transfer_inventory", aliases=["invtransfer", "auction", "transferinv", "trinv"]
+    )
+    @commands.is_owner()
+    async def transfer_inventory(self, ctx: Ctx, from_user: discord.User, to_user: discord.User):
```

Comment:
Can you make this take `int | discord.User` so that this can handle users which don't share any servers with the bot?